README.md

```diff
@@ -87,7 +87,7 @@ class ColorChanger(App):
             self.background = f"on color({event.key})"
 
 
-ColorChanger.run(log="textual.log")
+ColorChanger.run(log_path="textual.log")
```

Comment:
just the result of the renaming of `log`to `log_path` (and `css_file` to `css_path`) in the App constructor :slightly_smiling_face: